tools/state-copy/remove.mjs

```diff
@@ -0,0 +1,58 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { Keyring } from "@polkadot/keyring";
+
+const keyring = new Keyring({ type: "sr25519" });
+
+async function getKeys(api, storageKey) {
+  const pageSize = 500;
+  const result = [];
+  let startKey = "";
+  while (true) {
+    const page = await api.rpc.state.getKeysPaged(storageKey, pageSize, startKey);
+    result.push(...page.map((x) => x.toString()));
+    if (page.length === 0) {
+      break;
+    }
+
+    startKey = page[page.length - 1].toString();
+  }
+  return result;
+}
+
+export async function remove(destUrl, storageKey, sudoUri = "//Alice", filterKeys = []) {
```

Comment:
nit: removeStorageKeys would be better IMO

Reply:
```suggestion
export async function removeStorageKeys(destUrl, storageKey, sudoUri = "//Alice", filterKeys = []) {
```